docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3926,6 +3926,7 @@ where:
         - `adguard_ext_opera` — AdGuard Browser Extension for Opera
         - `adguard_ext_android_cb` — AdGuard Content Blocker for mobile Samsung and Yandex browsers
         - `ext_ublock` — special case; this one is declared when a uBlock version of a filter is compiled by the [FiltersRegistry]
+        - `cap_html_filtering` - products that have HTML filtering capability - AdGuard for Windows, Mac and Android
```

Comment:
it's not supported by tsurlfilter or filters-downloader, is it? is there any task for that?

Reply:
It may be supported by AdGuard Browser Extension for Firefox since it does HTML filtering.

The task: https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2259

Reply:
```suggestion
        - `cap_html_filtering` — products that support HTML filtering rules: AdGuard for Windows, AdGuard for Mac, and AdGuard for Android
```